pkgs/applications/networking/mullvad-vpn/default.nix

```diff
@@ -63,7 +63,7 @@ stdenv.mkDerivation rec {
 
   unpackPhase = "dpkg-deb -x $src .";
 
-  runtimeDependencies = [ (lib.getLib systemd) libnotify libappindicator ];
+  runtimeDependencies = [ (lib.getLib systemd) libnotify libappindicator wayland ];
```

Comment:
Shouldn't it be easier to put this into `deps` such that autopatchelf just links it in?

Reply:
simply putting it in `deps` didn't work, is there anything else that I would have needed to do for autopatchelf?

Reply:
I see. I actually have no experience with patchelf, I just assumed it'd work like that.

Reply:
`runtimeDependencies` is the correct way to handle a case where the lib is not listed in the elf file, but is `dlopen`ned after link/loading sometime.